### PR TITLE
Add documentation to develop

### PR DESCRIPTION
**Pull request by mimischi**
This is convoluted, but we also want to have the documentation in `develop`, before merging it into `master`.

For all discussions see PR #88. Replaces #100.

### CODE REVIEW

**Comment by mimischi** [comment]
Something went wrong with #100. Testing changes here.

**Comment by mimischi** [comment]
@kain88-de Any pointers on why the OS X job is failing?

https://api.travis-ci.org/v3/job/440162835/log.txt

File: .travis.yml [hunk 1]
```diff
@@ -60,15 +61,18 @@ jobs:
...
     - stage: "macOS - Unit tests"
       language: generic
       name: "Python 2.7"
       env: PYTHON_VERSION=2.7
-      install:
-        - git clone git://github.com/astropy/ci-helpers.git
-        - source ci-helpers/travis/setup_conda.sh
-        - pip install -e .
-      script: $MAIN_CMD $SETUP_CMD
+      install: pip install . mdbenchmark -v
```
Comment (kain88-de) [inline .travis.yml]: This build runs now on ubuntu and *not* osx. Could this be part of the problem? You sed the PYTHON_VERSION environment variable but the global used version is 3.6 set at the top of the file.

Comment (mimischi) [reply .travis.yml]: Wow, that makes sense. Solving month-old merge conflicts is a pain. Thanks for the heads up!

**Pull request merged**